RockPaperScissors/RockPaperScissors.cs

```diff
@@ -14,12 +14,62 @@ public static void Main()
 
             // leave this command at the end so your program does not close automatically
             Console.ReadLine();
+            
+        
         }
         
         public static string CompareHands(string hand1, string hand2)
         {
             // Your code here
+            if (hand1 == "rock" && hand2 == "rock")
```

Comment:
This method directly prints out to the console, I would like for this method to not do any user interaction, but return the appropriate value/result. And the calling method (Main in your case) handle the user interaction.